class7-week3/models/todo.js

```diff
@@ -64,17 +76,100 @@ class Todo {
 
   remove(id, callback) {
     this.load((error, todos) => {
-      if (error) { callback(error); return }
+      if (error) {
+        callback(error);
+        return
+      }
 
       todos = todos.filter(t => t.id !== id)
 
       this.save(todos, error => {
-        if (error) { callback(error); return }
+        if (error) {
+          callback(error);
+          return
+        }
+        callback()
+      })
+    })
+  }
+
+  clean(callback) {
+    this.load((error, todos) => {
+      if (error) {
+        callback(error);
+        return
+      }
+
+todos = todos.filter(t => t.id === '')
```

Comment:
Is this line necessary? What is a simpler way that we could clear out the list of todos before saving?